lib/ht_sip_validator/validator/ocr/well_formed_xml.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+require "nokogiri"
+
+# frozen_string_literal: true
+module HathiTrust::Validator
+
+  # Validates that provided coordinate OCR is UTF-8
+  class OCR::WellFormedXML < FileValidator
+
+    def perform_file_validation(filename, filehandle)
+      check_well_formed_xml(filename, filehandle)
+    end
+
+    def should_validate?(filename)
+      !filename.match(/\.(html|xml)$/).nil?
```

Comment:
I was unaware we were accepting html.  If so, this should be well formed markup validation, since html isn't a subset of xml.